examples/basic_example_open_menu.yaml

```diff
@@ -0,0 +1,98 @@
+# This file demonstrates the most basic usage of webviz in a FMU setting
+# The configuration files uses YAML (https://en.wikipedia.org/wiki/YAML).
+
+title: Reek Webviz Demonstration
+
+options:
+  menu:
+    initially_pinned: True
+
+layout:
+
+ - page: Front page
+   content:
+    - BannerImage:
+        image: ./example_banner.png
+        title: My banner image
+    - Webviz created from configuration file.
+    - Some other text, potentially with strange letters like Åre, Smørbukk Sør.
+
+ - page: Markdown example
+   content:
+    - Markdown:
+        markdown_file: ./example-markdown.md
+
+ - page: Table example
+   content:
+    - DataTable:
+        csv_file: ./example_data.csv
+
+ - page: PDF example
+   content:
+    - EmbedPdf:
+        pdf_file: ./example.pdf
+
+ - page: Syntax highlighting example
+   content:
+    - SyntaxHighlighter:
+        filename: ./basic_example.yaml
+
+ - page: Plot a table
+   content:
+    - TablePlotter:
+        csv_file: ./example_data.csv
+        # Everything below are examples of optional settings
+        filter_cols:
+          - Well
+          - Segment
+          - Average permeability (D)
+        plot_options:        
+          type: bar
+          facet_col: Well
+          color: Segment
+          barmode: group
+        filter_defaults:
+          Well:
+            - A-1H
+            - A-2H
+            - C-1H
+        column_color_discrete_maps:
+        # Supports css color codes, rgb and hex code.
+        # Note that hex code needs quotes '' to not be read as a comment
+          Segment:
+            A: '#ff0000'
+            B: rgb(0,255,0)
+            C: blue
+        contact_person:
+          name: Ola Nordmann
+          phone: +47 12345678
+          email: some@email.com
+
+ - page: Plot a table (locked)
+   content:
+    - TablePlotter:
+        csv_file: ./example_data.csv
+        lock: true
+        plot_options:
+          x: Well
+          y: Initial reservoir pressure (bar)
+          size: Average permeability (D)
+          facet_col: Segment
+        contact_person:
+          name: Kari Nordmann
+          phone: 12345678
+          email: someother@email.com
+
+ - page: Pivot Table
+   content:
+    - PivotTable:
+        csv_file: ./example_data.csv
+        options:
+          cols:
+            - Well
+          rows:
+            - Segment
+          vals:
+            - Average permeability (D)
+          aggregatorName: Average
+          rendererName: Table Heatmap
```

Comment:
Remove this now that the selenium test issue is solved?

Reply:
Ah, ok. We could perhaps adjust the other example? No strong opinions, but it would be nice to not maintaining two examples that are almost identical.